rambo/options.py

```diff
@@ -4,6 +4,19 @@
 from rambo.settings import SETTINGS, PROJECT_LOCATION, PROJECT_NAME
 from rambo.utils import get_env_var, set_env_var
 
+def address_option(address=None):
+    '''Validate address. If "dhcp", get an address in 172.28 network. If not supplied, set to default 192.168.33.10. Set as env var.
```

Comment:
There's no validation happening, really. We should check if it's the string 'dhcp', or somehow looks like an ip address. That's probably a googlable thing.

Reply:
https://docs.python.org/3/library/ipaddress.html

Reply:
It looks like Vagrant can accept an ipv4 or ipv6 address with the same syntax, which is nice. https://www.vagrantup.com/docs/networking/private_network.html So we can check for it being either with the above link.